ovh/data_iam_reference_action.go

```diff
@@ -0,0 +1,82 @@
+package ovh
+
+import (
+	"context"
+	"log"
+	"net/url"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceIamReferenceActions() *schema.Resource {
+	return &schema.Resource{
```

Comment:
I am not sure if this and data_iam_reference_resource_kind should stay.
The data returned is not dependent on the account, but can change, with new actions added as the IAM is expanded, or new products are released.

I have trouble thinking of cases where they would be useful.

Reply:
If the API endpoint exists, then it legit to have it in the terraform provider :)
A Terraform provider should be a mirror on the remote API.
Let's keep it and see if it will use or not.